notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -0,0 +1,668 @@
+    "parser.add_argument('--data-size', type=str, dest='data_size', default=10, help='Movielens data size: 100k, 1m, 10m, or 20m')\n",
+    "args = parser.parse_args()\n",
+    "\n",
+    "data_pickle_path = os.path.join(args.data_folder, args.data_file)\n",
+    "\n",
+    "data = pd.read_pickle(path=data_pickle_path)\n",
+    "\n",
+    "train, test = python_random_split(data)\n",
+    "\n",
+    "# instantiate the SAR algorithm and set the index\n",
+    "header = {\n",
+    "    \"col_user\": \"UserId\",\n",
+    "    \"col_item\": \"MovieId\",\n",
+    "    \"col_rating\": \"Rating\",\n",
+    "    \"col_timestamp\": \"Timestamp\",\n",
+    "}\n",
+    "\n",
+    "logging.basicConfig(level=logging.DEBUG, \n",
+    "                    format='%(asctime)s %(levelname)-8s %(message)s')\n",
+    "\n",
+    "model = SARSingleNode(\n",
+    "    remove_seen=True, similarity_type=\"jaccard\", \n",
+    "    time_decay_coefficient=30, time_now=None, timedecay_formula=True, **header\n",
+    ")\n",
+    "\n",
+    "# train the SAR model\n",
+    "start_time = time.time()\n",
+    "\n",
+    "model.fit(train)\n",
+    "\n",
+    "train_time = time.time() - start_time\n",
+    "run.log(name=\"Training time\", value=train_time)\n",
+    "\n",
+    "start_time = time.time()\n",
+    "\n",
+    "top_k = model.recommend_k_items(test)\n",
+    "\n",
+    "test_time = time.time() - start_time\n",
+    "run.log(name=\"Prediction time\", value=test_time)\n",
+    "\n",
+    "# TODO: remove this call when the model returns same type as input\n",
+    "top_k['UserId'] = pd.to_numeric(top_k['UserId'])\n",
+    "top_k['MovieId'] = pd.to_numeric(top_k['MovieId'])\n",
+    "\n",
+    "# evaluate\n",
+    "eval_map = map_at_k(test, top_k, col_user=\"UserId\", col_item=\"MovieId\", \n",
+    "                    col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                    relevancy_method=\"top_k\", k=args.top_k)\n",
+    "eval_ndcg = ndcg_at_k(test, top_k, col_user=\"UserId\", col_item=\"MovieId\", \n",
+    "                      col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                      relevancy_method=\"top_k\", k=args.top_k)\n",
+    "eval_precision = precision_at_k(test, top_k, col_user=\"UserId\", col_item=\"MovieId\", \n",
+    "                                col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                                relevancy_method=\"top_k\", k=args.top_k)\n",
+    "eval_recall = recall_at_k(test, top_k, col_user=\"UserId\", col_item=\"MovieId\", \n",
+    "                          col_rating=\"Rating\", col_prediction=\"prediction\", \n",
+    "                          relevancy_method=\"top_k\", k=args.top_k)\n",
+    "\n",
+    "run.log(\"map\", eval_map)\n",
+    "run.log(\"ndcg\", eval_ndcg)\n",
+    "run.log(\"precision\", eval_precision)\n",
+    "run.log(\"recall\", eval_recall)\n",
+    "\n",
+    "os.makedirs('outputs', exist_ok=True)\n",
```

Comment:
you don't need to create this directory since `./outputs` should already exist. If it doesn't then its probably a bug.